packages/salesforcedx-utils-vscode/src/i18n/localization.ts

```diff
@@ -61,7 +61,7 @@ export class Message implements LocalizationProvider {
     }
 
     if (args.length >= 1) {
-      const expectedNumArgs = possibleLabel.split('%s').length - 1;
+      const expectedNumArgs = possibleLabel.split(/%[sdifjoO]/).length - 1;
```

Comment:
No, this was intentional and I would like to keep it. The function used to fill in values for messages is using nodejs util.format. It exposes a number of of format options, where `%s` is one of many. This change allows devs to use type specific formats, rather than relying on type->string transformation. I only added formats I thought might be usefull for our work in extension messages.

See https://nodejs.org/api/util.html#utilformatformat-args for details.

Reply:
cool works for me. Can we add a code comment about what it accomplishes?   I get it now...the message can contain %s or %d or %i or %f etc.